pages/first-project.mdx

```diff
@@ -70,17 +71,28 @@ When you set up a project, you’re defining the tools and processes that transf
 
 Once your project is created, configure the AI tools it will use. Configurations include:
 
-- **Vectorizers**: Convert data into embeddings for tasks like semantic search.
-- **Enhancers**: Enrich data by summarizing, tagging, or extracting metadata.
-- **Generative Modules**: Enable AI-driven content creation.
+- [**Vectorizers**](/project-configurations/vectorizers): Convert data into embeddings for tasks like [semantic search](/graphql-api/search/semantic-search).
+- [**Enhancers**](/project-configurations/enhancers): Enrich data by summarizing, tagging, or extracting metadata.
+- [**Generative Modules**](/project-configurations/generative-modules): Enable AI-driven content creation.
 
-For this guide, we use a **vectorizer** for processing text and two built in enhancers: **AutoVision** for enhancing images and **AutoSummary** for text documents.
+For this guide, we use a [**vectorizer**](/admin-api#textvectorizer-configuration) for processing text and two built in enhancers: [**AutoVision**](/admin-api#autovision-configuration) for enhancing images and [**AutoSummary**](/admin-api#autosummary-configuration) for [text documents](/graphql-api/data-types/textdocument).
 
 <Tabs items={['Dashboard', 'SDK']}>
   <Tabs.Tab>
-  You’ll find these options in the [**Project Configuration**](/project-configurations/vectorizers) section during project setup.
+  You’ll find these options in the [**Project Configuration**](/project-configurations/vectorizers) -> `ADVANCED` section during project setup.
+
+  - Configure `Text Vectorizers` to enable semantic search and text processing
+   ![Setting up text vectorizer](/images/docs/get-started/setup-text-vectorizer.png)
+    
+  - Set up `Generative Search` with your preferred model
```

Comment:
Not preferable model!  we need to mention the exact name of model like how it's in SDK example.